src/main/java/HW_7/Circle.java

```diff
@@ -0,0 +1,23 @@
+package HW_7;
+
+public class Circle extends Figure {
+    private double radius;
+    public static final double PI = 3.14;
+    public Circle(double radius){
```

Comment:
there is should an extra line between 5 and 6 line

Reply:
fixed

Reply:
not fixed

Reply:
fixed